packages/react-ui-components/src/utils/useFontsApi.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useMemo } from 'react';
+
+const FONT_NOT_FOUND_ERROR = `Font was not found, make sure the font is included in your css`;
+
+export const useFontsApi = (fontFamily: string, fontSize: number) => {
+	const fontName = useMemo(() => `${fontSize}px ${fontFamily}`, [fontFamily, fontSize]);
```

Comment:
Since we are loading these properties from a callback, we should not memo this. We should build this string inside the loadFont.